Kiam 2.7 doesn't build

Steps:
`git clone git@github.com:uswitch/kiam.git --branch 2.7-deploy`
` cd kiam`
` make`

When running `make` in the project root, this is the ouput:

```
# github.com/uswitch/kiam/pkg/server
pkg/server/server.go:187: cannot use server (type *KiamServer) as type kiam.KiamServiceServer in argument to ServerWithTelemetry:
	*KiamServer does not implement kiam.KiamServiceServer (wrong type for GetHealth method)
		have GetHealth("context".Context, *kiam.GetHealthRequest) (*kiam.HealthStatus, error)
		want GetHealth("github.com/uswitch/kiam/vendor/golang.org/x/net/context".Context, *kiam.GetHealthRequest) (*kiam.HealthStatus, error)
pkg/server/telemetry.go:30: cannot use TelemetryClient literal (type *TelemetryClient) as type kiam.KiamServiceClient in return argument:
	*TelemetryClient does not implement kiam.KiamServiceClient (wrong type for GetHealth method)
		have GetHealth("context".Context, *kiam.GetHealthRequest, ...grpc.CallOption) (*kiam.HealthStatus, error)
		want GetHealth("github.com/uswitch/kiam/vendor/golang.org/x/net/context".Context, *kiam.GetHealthRequest, ...grpc.CallOption) (*kiam.HealthStatus, error)
pkg/server/telemetry.go:71: cannot use TelemetryServer literal (type *TelemetryServer) as type kiam.KiamServiceServer in return argument:
	*TelemetryServer does not implement kiam.KiamServiceServer (wrong type for GetHealth method)
		have GetHealth("context".Context, *kiam.GetHealthRequest) (*kiam.HealthStatus, error)
		want GetHealth("github.com/uswitch/kiam/vendor/golang.org/x/net/context".Context, *kiam.GetHealthRequest) (*kiam.HealthStatus, error)
make: *** [bin/agent] Error 2
```

--- a/README.md
+++ b/README.md
@@ -57,8 +57,9 @@ This is the process that would typically be deployed as a DaemonSet to ensure th
 This process is responsible for connecting to the Kubernetes API Servers to watch Pods and communicating with AWS STS to request credentials. It also maintains a cache of credentials for roles currently in use by running pods- ensuring that credentials are refreshed every few minutes and stored in advance of Pods needing them.
 
 ## Building locally
-If you want to build and run locally you can
-
+If you want to build and run locally:
+- `go version` >= 1.9
+- run the following
 ```
 $ mkdir -p $GOPATH/src/github.com/uswitch
 $ git clone git@github.com:uswitch/kiam.git $GOPATH/src/github.com/uswitch/kiam

